src/build/packages.rs

```diff
@@ -288,14 +306,15 @@ fn read_dependencies<'a>(
             let dependencies = read_dependencies(
                 &mut registered_dependencies_set.to_owned(),
                 &bsconfig,
+                &canonical_path,
                 project_root,
                 workspace_root.to_owned(),
             );
 
             Dependency {
                 name: package_name.to_owned(),
                 bsconfig,
-                path,
+                path: canonical_path,
```

Comment:
Ideally to start storing both canonical and human readable paths in the future, to use it for more human readable compilation errors.

Reply:
hmm yeah! But in some cases the canonical is the more user readable one (monorepo paths that are symlinked in node_modules)